### PR TITLE
Bootstrap autoapprover: use oc_obj to apply objects

### PR DESCRIPTION
**Pull request by vrutkovs**

### CODE REVIEW

**Comment by vrutkovs** [comment]
bot, retest this please

File: roles/openshift_bootstrap_autoapprover/tasks/main.yml [hunk 1]
```diff
@@ -18,15 +18,58 @@
...
+    name: "node"
+    kind: ImageStream
+    state: absent
+    namespace: openshift-infra
+
+- name: Create imagestreamtag
+  oc_obj:
+    state: present
+    kind: ImageStreamTag
+    name: "node:v3.10"
```
Comment (DanyC97) [inline roles/openshift_bootstrap_autoapprover/tasks/main.yml]: why the colon @vrutkovs  ?

Comment (vrutkovs) [reply roles/openshift_bootstrap_autoapprover/tasks/main.yml]: That's an ImageStreamTag, created in https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_bootstrap_autoapprover/files/openshift-bootstrap-images.yaml#L4

Comment (DanyC97) [reply roles/openshift_bootstrap_autoapprover/tasks/main.yml]: ah okay, thanks

**Comment by vrutkovs** [comment]
Master branch is closed! A major refactor is ongoing in devel-40. Changes for 3.x should be made directly to the latest release branch they're relevant to and backported from there.

**Pull request closed without merging**